src/scss/global.scss

```diff
@@ -6,6 +6,7 @@
 @import "node_modules/bootstrap/scss/bootstrap";
 @import "node_modules/bootswatch/dist/darkly/bootswatch";
 @import "./constants.scss";
+@import "./ClientCustom/Custom.scss";
```

Comment:
This needs to be updated based on the renamed file

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1647/commits/2b61b8cc3f4f16c3310a72d0e2059b7c070d091f